model/atmosphere/dycore/tests/dycore_tests/conftest.py

```diff
@@ -38,5 +43,155 @@
     savepoint_velocity_init,
     step_date_exit,
     step_date_init,
+    stretch_factor,
+    top_height_limit_for_maximal_layer_thickness,
     vn_only,
 )
+
+
+def construct_interpolation_state(
```

Comment:
I would actually not put anything into `conftest.py`that is not handled by pytest. Meaning I would not put normal test utitility functions there, that are not fixtures.

So for these functions here, I would simply revert the "old" utils.py and use them from there inside the dycore tests, unless you have a good reason to put them into `conftest.py`?

Reply:
Ok, I have moved these utility functions back to `utils.py`